packages/@lwc/integration-karma/test/signal/reactivity/index.spec.js

```diff
@@ -0,0 +1,94 @@
+import { createElement } from 'lwc';
+
+import Reactive from 'x/reactive';
+import NonReactive from 'x/nonReactive';
+import ExplicitSubscribe from 'x/explicitSubscribe';
+import List from 'x/list';
+
+import { Signal } from 'x/signal';
+
+const createElementSignalAndInsertIntoDom = async (tagName, ctor, signalInitialValue) => {
+    const elm = createElement(tagName, { is: ctor });
+    const signal = new Signal(signalInitialValue);
+    elm.signal = signal;
+    document.body.appendChild(elm);
+    await Promise.resolve();
+
+    return { elm, signal };
+};
+
+describe('signal reaction in lwc', () => {
+    it('should render signal value', async () => {
```

Comment:
These tests validate the content is displayed correctly after the framework has subscribed to a signal.